package.json

```diff
@@ -67,6 +67,7 @@
     "d3-dsv": "^3.0.1",
     "html-to-text": "^9.0.5",
     "langchain": "^0.2.0",
+    "langsmith": "^0.1.41",
```

Comment:
I think we can make langsmith an optional dependency? If they want to use the integration, they will have to install the package too

Reply:
Tbh, makes sense. I'll see what I can do.